src/test/java/racingcar/RacingCarGameTest.java

```diff
@@ -0,0 +1,39 @@
+package racingcar;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+class RacingCarGameTest {
+    private static final int CAR_MOVE_THRESHOLD = 4;
+
+    @Test
+    @DisplayName("자동차들이 최대 입력된 시도 회수만큼 움직여야 한다")
+    void process() {
+        List<Car> carList = new ArrayList<>();
+        carList.add(new Car());
+        carList.add(new Car());
+        carList.add(new Car());
+
+        int tryCount = 3;
+        new RacingCarGame(carList, tryCount, new ResultView()).startGame(new RandomForTest(CAR_MOVE_THRESHOLD));
+        Assertions.assertThat(carList).allMatch(car -> car.getPosition() == tryCount);
```

Comment:
```suggestion
        new RacingCarGame(carList, tryCount, new ResultView()).startGame(new RandomForTest(CAR_MOVE_THRESHOLD));
        Assertions.assertThat(carList).allMatch(car -> car.getPosition() == tryCount);
```

상태를 가진 객체에게 물어보거나 객체간의 비교를 한다면 더 좋을것 같은데, 이후 단계에서 함께 고민해보면 좋겠네요 😄